tests/AbstractSerializerTest.php

```diff
@@ -545,4 +545,9 @@ class SerializerTestObject
     private $foo = 'bar';
 
     public $key = 'value';
+
+    public static function testy(): array
+    {
+        return [];
```

Comment:
Throw an exception here: we should be sure of not invoking it while serializing it.

Reply:
What's the purpose of this function having a return type that is not `void`?